app/scripts/metamask-controller.js

```diff
@@ -6652,12 +6652,31 @@ export default class MetamaskController extends EventEmitter {
    * @param {string} origin - the domain to safelist
    */
   safelistPhishingDomain(origin) {
+    this.metaMetricsController.trackEvent({
+      category: MetaMetricsEventCategory.Phishing,
+      event: MetaMetricsEventName.ProceedAnywayPhishingPage,
+      properties: {
+        origin,
```

Comment:
```suggestion
        url: origin,
        referrer: {
            url: origin,
        },
```

Present day there are several events which do not line up their properties correctly. `PermissionsApproved` (and all Confirmations) for example use the `referrer: { url: origin }` pattern, but `PhishingPageDisplayed` uses `url` as a top level field which makes them not line up when creating funnels.

This way we can finally make these events consistent.

Reply:
If you could also add the above suggestion for `referrer` to the `PhishingPageDisplayed` event it would be very helpful for me

Reply:
Love this suggestion! <3